src/lib/host/aws.js

```diff
@@ -114,12 +114,7 @@ module.exports = {
 
     let userdata = await getJsonData(`${baseUrl}/user-data`);
     let securityToken = userdata.securityToken;
-
-    let rootUrl = userdata.rootUrl;
-    if (!rootUrl) {
-      // for backward compatibility, assume the legacy instance if no rootUrl is given
-      rootUrl = 'https://taskcluster.net';
-    }
+    let rootUrl = userdata.taskclusterRootUrl;
```

Comment:
What if `taskclusterRootUrl` isn't defined?

Reply:
Is an assert the best way to bail out at this point?

Reply:
Ah, I see a call to `shutdown -h now` a few lines later..

Reply:
In a few months, that won't work, though.  Does it make sense at that point to have a default value that doesn't work?

Reply:
Fair enough :)